tests/resources/downloads/__init__.py

```diff
@@ -0,0 +1,5 @@
+#
```

Comment:
There's likely a cleaner way than this: use pytest built-in functionality for temporary directories.

Details: https://docs.pytest.org/en/stable/tmpdir.html

Then you won't need tests/resources/downloads, or adding the extra .gitignore line.

Reply:
thank you, Trent! I suggested this implementation and did a similar thing on the 3rd code coverage batch. I've fixed it there. @mariacarmina sorry for misleading you, Trent's suggestion is much better.